content/zh-cn/docs/concepts/security/security-checklist.md

```diff
@@ -0,0 +1,749 @@
+镜像扫描应在将容器镜像部署到集群之前完成，通常作为 CI/CD 流水线中的部署过程的一部分来完成。
+镜像扫描的目的是获取有关容器镜像中可能存在的漏洞及其预防措施的信息，
+例如使用[公共漏洞评分系统 （Common Vulnerability Scoring System，CVSS）](https://www.first.org/cvss/)评分。
+如果镜像扫描的结果与管道合性规则匹配，则只有经过正确修补的容器镜像才会最终进入生产环境。
+
+<!--
+## Admission controllers
+
+- [ ] An appropriate selection of admission controllers is enabled.
+- [ ] A pod security policy is enforced by the Pod Security Admission or/and a
+  webhook admission controller.
+- [ ] The admission chain plugins and webhooks are securely configured.
+-->
+## 准入控制器 {#admission-controllers}
+
+- [ ] 选择启用适当的准入控制器。
+- [ ] Pod 安全策略由 Pod 安全准入强制执行，或者和 Webhook 准入控制器一起强制执行。
+- [ ] 保证准入链插件和 Webhook 的配置都是安全的。
+
+<!--
+Admission controllers can help to improve the security of the cluster. However,
+they can present risks themselves as they extend the API server and
+[should be properly secured](/blog/2022/01/19/secure-your-admission-controllers-and-webhooks/).
+-->
+准入控制器可以帮助提高集群的安全性。
+然而，由于它们是对 API 服务器的扩展，其自身可能会带来风险，
+所以它们[应该得到适当的保护](/blog/2022/01/19/secure-your-admission-controllers-and-webhooks/)。
+
+<!--
+The following lists present a number of admission controllers that could be
+considered to enhance the security posture of your cluster and application. It
+includes controllers that may be referenced in other parts of this document.
+-->
+下面列出了一些准入控制器，可以考虑用这些控制器来增强集群和应用程序的安全状况。
+列表中包括了可能在本文档其他部分曾提及的控制器。
+
+<!--
+This first group of admission controllers includes plugins
+[enabled by default](/docs/reference/access-authn-authz/admission-controllers/#which-plugins-are-enabled-by-default),
+consider to leave them enabled unless you know what you are doing:
+-->
+第一组准入控制器包括[默认启用的插件](/zh-cn/docs/reference/access-authn-authz/admission-controllers/#which-plugins-are-enabled-by-default)，
+除非你知道自己在做什么，否则请考虑保持它们处于被启用的状态：
+
+<!--
+[`CertificateApproval`](/docs/reference/access-authn-authz/admission-controllers/#certificateapproval)
+: Performs additional authorization checks to ensure the approving user has
+permission to approve certificate request.
+-->
+[`CertificateApproval`](/zh-cn/docs/reference/access-authn-authz/admission-controllers/#certificateapproval)
+: 执行额外的授权检查，以确保审批用户具有审批证书请求的权限。
+
+<!--
+[`CertificateSigning`](/docs/reference/access-authn-authz/admission-controllers/#certificatesigning)
+: Performs additional authorization checks to ensure the signing user has
+permission to sign certificate requests.
+-->
+[`CertificateSigning`](/zh-cn/docs/reference/access-authn-authz/admission-controllers/#certificatesigning)
+: 执行其他授权检查，以确保签名用户具有签名证书请求的权限。
+
+<!--
+[`CertificateSubjectRestriction`](/docs/reference/access-authn-authz/admission-controllers/#certificatesubjectrestriction)
+: Rejects any certificate request that specifies a 'group' (or 'organization
+attribute') of `system:masters`.
+-->
+[`CertificateSubjectRestriction`](/zh-cn/docs/reference/access-authn-authz/admission-controllers/#certificatesubjectrestriction)
+: 拒绝将 `group` （或`organization attribute`）设置为 `system:masters` 的所有证书请求。
```

Comment:
```suggestion
: 拒绝将 `group` （或 `organization attribute`）设置为 `system:masters` 的所有证书请求。
```

Reply:
@zhangxyjlu I don't think this is a blocking issue because in the rendered output, there is already a visible space even if the space is not there. What do you think?

Reply:
![image](https://user-images.githubusercontent.com/8384921/199708961-38cd6f28-9797-456a-8097-9ade022a27f5.png)

在界面上对比了一下，原本是有空白的，加上一点也不错。
不过去掉左括号左边的空白看上去更均衡一些，请两位参考